pym/portage/sync/modules/git/git.py

```diff
@@ -130,3 +130,15 @@ def update(self):
 			cwd=portage._unicode_encode(self.repo.location))
 
 		return (os.EX_OK, current_rev != previous_rev)
+
+	def retrieve_head(self, **kwargs):
+		'''Get information about the head commit'''
+		if kwargs:
+			self._kwargs(kwargs)
```

Comment:
Well, self._kwargs is also useful if some other method wants to call retrieve_head after it has already called self._kwargs earlier.